src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/ApplicationClosingDateValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Esfa.Vacancy.Domain.Validation;
+using FluentValidation;
+
+namespace Esfa.Vacancy.Application.Commands.CreateApprenticeship.Validators
+{
+    public class ApplicationClosingDateValidator : AbstractValidator<DateTime>
+    {
+        public ApplicationClosingDateValidator()
+        {
+            RuleFor(closingDate => closingDate)
+                .GreaterThanOrEqualTo(DateTime.Today.AddDays(1))
```

Comment:
Can you stick the `DateTime.Today.AddDays(1)` in a variable.

Reply:
Or simply use GreaterThan(DateTime.Today)

Reply:
DateTime.Today won't work. E.g. DateTime.Today.AddSeconds(1) is greater than today but not what the validation rule should allow.